packages/utils/src/normalize.ts

```diff
@@ -1,15 +1,25 @@
 /**
- * Normalize a value between a range
+ * The `normalize` function scales a value within a specific range to a
+ * normalized range between `0` and `1`. Given an input value and a range
+ * (`min` to `max`), it calculates the proportion of the value relative to the
+ * range. This is useful for standardizing data, making it easier to work with
+ * values across different scales.
  *
- * @param value - The value to normalize
- * @param min - The minimum value of the range
- * @param max - The maximum value of the range
- * @returns The normalized value
+ * @param value - The number to be normalized within the range.
+ * @param min - The minimum value of the range.
+ * @param max - The maximum value of the range.
```

Comment:
**thought:** why not "number" here as well?

Reply:
Updated copy to `value` as this text is not directly tied to it's ts type.